src/utils/slack.js

```diff
@@ -17,9 +17,13 @@ export const loadUsers = () => {
 };
 
 export const sendSlackMessage = async (id, itemName, storeCode) => {
-	await fetch(`http://slack.com/api/chat.postMessage?token=${
-		this.props.token
-	}&
-	channel=${id}&
-	text=${`Click to purchase your ${itemName}: https://honesty.store/item/${storeCode}`}`);
+	try {
+		await fetch(`http://slack.com/api/chat.postMessage?token=${
+			this.props.token
+		}&
+		channel=${id}&
+		text=${`Click to purchase your ${itemName}: https://honesty.store/item/${storeCode}`}`);
+	} catch (error) {
+		console.error('Failed to obtain env variable: SLACK_TOKEN');
```

Comment:
Could this also fail for other issues than just the token? E.g. network?

Should we let the caller deal with the error (dispatching an error action) and warn in the UI if there was no slack token detected?